app/src/main/java/com/example/mytestproject/ui/MainActivity.kt

```diff
@@ -42,14 +42,25 @@ class MainActivity : AppCompatActivity(){
 
         navController.addOnDestinationChangedListener { _, destination, _ ->
 
-            bottom_navigation.visibility =  when (destination.id) {
-                R.id.nav_second_fragment,R.id.nav_third_fragment, R.id.search_city ->  View.GONE
-                else ->  View.VISIBLE
+            bottom_navigation.visibility = when (destination.id) {
+                R.id.nav_second_fragment, R.id.nav_third_fragment, R.id.search_city -> View.GONE
+                else -> View.VISIBLE
             }
         }
     }
 
     override fun onSupportNavigateUp(): Boolean {
         return navController.navigateUp(appBarConfiguration) || super.onSupportNavigateUp()
     }
+
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
```

Comment:
This menu will be shown on fragments Second and Third, probably. But it shouldn't.

Reply:
Fixed